package/MDAnalysis/topology/HoomdXMLParser.py

```diff
@@ -121,12 +121,11 @@ def parse(self, **kwargs):
         ):
             try:
                 val = configuration.find(attrname)
-                vals = [mapper(el) for el in val.text.strip().split()]
+                vals = [mapper(el) for el in val.text.strip().split('\n')]
```

Comment:
Wow that fixed it? What was wrong with splitting on any white space? I thought the problem was not doing `.split()[1:]`

Reply:
That wasn't the issue. The issued ended up being with the plural vs singular terms for mass and charge being called at the wrong time.